### PR TITLE
Added C++ elastic lib

### PR DESCRIPTION
**Pull request by DanPeled**
Might be a good idea to use JNI with the java version if it is planned to move this into a vendordep

### CODE REVIEW

**Comment by Cynosure-null** [comment]
Hi, I've been trying to use this for my team and I've been running into the error `fatal error: networktables/StringPublisher.h: No such file or directory`. on line 6 What WPILib version does this work on? Thanks for your work on this.

File: elasticlib/C++/include/Elastic.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <string>
+#include <iostream>
+#include <networktables/NetworkTableInstance.h>
+#include <networktables/StringPublisher.h>
```
Comment (spacey-sooty) [inline elasticlib/C++/include/Elastic.h:6]:
```suggestion
```

**Review by spacey-sooty — CHANGES_REQUESTED** [review]

**Comment by spacey-sooty** [comment]
> Hi, I've been trying to use this for my team and I've been running into the error `fatal error: networktables/StringPublisher.h: No such file or directory`. on line 6 What WPILib version does this work on? Thanks for your work on this.

StringPublisher.h doesn't exist. If you remove it this should work

**Comment by Cynosure-null** [comment]
I don't have write access to this pr, but line 12 should be `        nt::StringPublisher publisher = topic.Publish({.sendAll = true, .keepDuplicates = true});` to prevent issues about not finding the `nt::PubSubOptions` class

**Comment by DanPeled** [comment]
I'll see what I did wrong when I get the chance, probably will get it done by the end of the day

**Comment by DanPeled** [comment]
Depends on #133

**Pull request merged**